accounts-db/src/accounts_index.rs

```diff
@@ -3362,6 +3362,10 @@ pub mod tests {
         );
     }
 
+    fn make_empty_token_account_data() -> Vec<u8> {
+        vec![0; solana_spl_ids::token::Account::get_packed_len()]
+    }
```

Comment:
Would have been nice to have this refactoring (and SPL_TOKENS) in a separate commit